src/Symfony/Component/Process/Process.php

```diff
@@ -468,8 +474,6 @@ public function getIncrementalErrorOutput()
     public function getExitCode()
     {
         if (!$this->enhanceSigchildCompatibility && $this->isSigchildEnabled()) {
-            $this->stop(0);
```

Comment:
these calls to `->stop(0)` have been added recently and never tagger but should be reverted